classes/options-general.php

```diff
@@ -485,7 +485,7 @@ class='howto'><?php echo $api_how_to; ?></span>
 
 													<tr valign="top">
 														<td><label
-																for="results_limit"><?php _e( 'Number of Results to Display', 'simplemap' ); ?></label>
+																for="results_limit"><?php _e( 'Number of Results to Display', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 151 characters

<!-- Comment by [SideCI](https://sideci.com) -->